packages/core/src/connectivity/scp-cf/destination/destination-accessor-proxy-configuration.spec.ts

```diff
@@ -1,6 +1,6 @@
 import nock from 'nock';
 import {
-  mockedConnectivityServiceProxyConfig,
+  connectivityProxyConfigMock,
```

Comment:
General feedback: The renaming makes it hard to find the actual core of the PR. Consider doing such refactoring in a separate PR.